addons/website_slides/views/website_slides_templates_lesson_fullscreen.xml

```diff
@@ -135,13 +135,13 @@
                                 <div class="o_wslides_fs_slide_name text-600" t-esc="slide.name"/>
                             </div>
                         </span>
-                        <ul class="list-unstyled w-100 pt-2 small" t-if="slide.link_ids or slide.slide_resource_ids or (slide.question_ids and not slide.slide_type =='quiz')" >
-                            <li t-if="slide.link_ids" t-foreach="slide.link_ids" t-as="link" class="pl-0 mb-1">
-                                <a t-if="can_access" class="o_wslides_fs_slide_link" t-att-href="link.link" target="_blank">
-                                    <i class="fa fa-link mr-2"/><span t-esc="link.name"/>
+                        <ul class="list-unstyled w-100 pt-2 small" t-if="slide.slide_resource_ids or (slide.question_ids and not slide.slide_type =='quiz')" >
+                            <li t-if="slide.slide_resource_ids" t-foreach="slide.slide_resource_ids.filtered(lambda res: res.type == 'url')" t-as="resource" class="pl-0 mb-1">
```

Comment:
You check if `slide.slide_resource_ids` is not empty but you iterate on `slide.slide_resource_ids.filtered(...)`
So currently you might create an empty `<li>` 
```suggestion
                            <t t-set="resource_urls" t-value="slide.slide_resource_ids.filtered(lambda res: res.type == 'url')"/>
                            <li t-if="resource_urls" t-foreach="resource_urls" t-as="resource" class="pl-0 mb-1">
```